### PR TITLE
 fix: add zero padding to version timestamp

### PR DESCRIPTION
**Pull request by uday-rana**
Fixes https://github.com/planetarium/libplanet/issues/3955. I declared variables to store each formatted portion of the date for readability. Please let me know if you'd liked anything changed.

### CODE REVIEW

**Comment by uday-rana** [comment]
Re-creating [this PR](https://github.com/planetarium/libplanet/pull/3956) so it counts for Hacktoberfest. Thanks!

**Review by moreal — DISMISSED** [review]
LGTM 👍🏻 

------

We're now operating Modathon 2024 (Nine Chronicles Hackathon) with a prize pool about $10,000. If you're interested, please take a look at the following items 🙏🏻: 

- https://dorahacks.io/hackathon/planetarium-2024modathon/detail
- https://planetarium.dev/discord

File: scripts/determine-version.js [hunk 1]
```diff
@@ -118,11 +118,20 @@ async function main() {
...
+    const formattedHours = timestamp.getUTCHours().toString().padStart(2, "0");
+    const formattedMinutes = timestamp
+      .getUTCMinutes()
+      .toString()
+      .padStart(2, "0");
+    const formattedSeconds = timestamp
+      .getUTCSeconds()
+      .toString()
+      .padStart(2, "0");
+    const ts = `${timestamp.getUTCFullYear()}${formattedMonth}${formattedDate}${formattedHours}${formattedMinutes}${formattedSeconds}`;
```
Comment (s2quake) [inline scripts/determine-version.js:134]: It's not part of libplanet's lint settings, but is it possible to adjust this code to fit within 80 characters per line?

Comment (uday-rana) [reply scripts/determine-version.js]: Done, let me know how it looks (501e624)

Comment (s2quake) [reply scripts/determine-version.js]: > Done, let me know how it looks ([501e624](https://github.com/planetarium/libplanet/commit/501e6249f42d842a132897756d9fd2dd40b94858))

Thank you. However, when I save this code in an IDE like vscode, it reverts to the original code. So I think the following code is better, what do you think?

```js
    const ts =
      `${timestamp.getUTCFullYear()}` +
      `${formattedMonth}` +
      `${formattedDate}` +
      `${formattedHours}` +
      `${formattedMinutes}` +
      `${formattedSeconds}`;
```

Comment (uday-rana) [reply scripts/determine-version.js]: You're right. That also gives us room to do the formatting in-line, so we could remove the variable declarations:

```js
const timestamp = await getCommitTimestamp();
const ts =
  timestamp.getUTCFullYear() +
  (timestamp.getUTCMonth() + 1).toString().padStart(2, "0") +
  timestamp.getUTCDate().toString().padStart(2, "0") +
  timestamp.getUTCHours().toString().padStart(2, "0") +
  timestamp.getUTCMinutes().toString().padStart(2, "0") +
  timestamp.getUTCSeconds().toString().padStart(2, "0");
```

**Review by moreal — APPROVED** [review]

**Review by s2quake — APPROVED** [review]

**Pull request merged**